src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/AppointmentsService.kt

```diff
@@ -22,10 +25,14 @@ class AppointmentsService(
       prisoner.checkForDuplicateAppointment(AppointmentType.VLB_COURT_MAIN)
     }
 
+    val prison = prisonRepository.findByCode(prisoner.prisonCode)
+      ?.also { if (!user.isUserType(UserType.PRISON)) require(it.enabled) { "Prison with code ${it.code} is not enabled" } }
```

Comment:
Perhaps the error message could read "Prison with code ${it.code} is not enabled for self service" ?